src/context/store-context.js

```diff
@@ -149,38 +151,38 @@ export const StoreProvider = ({ children }) => {
       .addShippingMethod(state.cart.id, {
         option_id: id,
       })
-      .then(({ data }) => {
-        dispatch({ type: "setCart", payload: data.cart })
-        return data
+      .then(({ cart }) => {
+        dispatch({ type: "setCart", payload: cart })
+        return cart
       })
   }
 
   const createPaymentSession = async () => {
     return await client.carts
       .createPaymentSessions(state.cart.id)
-      .then(({ data }) => {
-        dispatch({ type: "setCart", payload: data.cart })
-        return data
+      .then(({ cart }) => {
+        dispatch({ type: "setCart", payload: cart })
+        return cart
       })
   }
 
   const completeCart = async () => {
     const data = await client.carts
       .complete(state.cart.id)
-      .then(({ data }) => data)
+      .then(({data}) => data)
```

Comment:
nit (non-blocking): I think the formatting is a bit inconsistent compared to other places in the same file. Perhaps prettier is not being picked up by your editor?